vm/systemSmartContracts/proto/delegation.proto

```diff
@@ -10,10 +10,10 @@ import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 message DelegationManagement {
   uint32 NumberOfContract     = 1 [(gogoproto.jsontag) = "NumberOfContracts"];
   bytes  LastAddress          = 2 [(gogoproto.jsontag) = "LastAddress"];
-  bytes  MinContractCreateFee = 3 [(gogoproto.jsontag) = "MinContractCreateFee", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
-  uint32 MinServiceFee        = 4 [(gogoproto.jsontag) = "MinServiceFee"];
-  uint32 MaxServiceFee        = 5 [(gogoproto.jsontag) = "MaxServiceFee"];
-  bytes  MinOperationValue    = 6 [(gogoproto.jsontag) = "MinOperationValue", (gogoproto.casttypewith) = "math/big.Int;github.com/ElrondNetwork/elrond-go/data.BigIntCaster"];
+  uint64 MinServiceFee        = 4 [(gogoproto.jsontag) = "MinServiceFee"];
```

Comment:
You have skipped an index. 3 here, and so on.

Reply:
done